### ISSUE TITLE
Can't find relationships between files

### ISSUE THREAD

**Issue by CyCle1024** [bkryza/clang-uml#40]
test.h:
```cpp
namespace test1 {

template <typename T>
class TestBase {
public:
  TestBase() = delete;
  ~TestBase() = default;
  TestBase(int a_, float b_, T c_) : a(a_), b(b_), c(c_) {};
  int a;
  float b;
  T c;
};

}
```

test.cc:
```cpp
#include <test.h>

namespace test1 {

template class TestBase<double *>;

class TestDerive : public TestBase<double *> {
public:
  TestDerive(int a_, float b_, double *c_);
};

TestDerive::TestDerive(int a_, float b_, double *c_) : TestBase<double *>(a_, b_, c_) {
  a += 1;
  b -= 1;
}

}
```

result test.puml:
```puml
@startuml
class "test1::TestBase<double*>" as C_0000000002
class C_0000000002 {
}
class "test1::TestDerive" as C_0000000011
class C_0000000011 {
{static} +TestDerive(int a_, float b_, double* c_) : void
}
class "test1::TestBase<T>" as C_0000000016
class C_0000000016 {
{static} +TestBase() : void
{static} +~TestBase() : void
{static} +TestBase(int a_, float b_, T c_) : void
+a : int
+b : float
+c : T
}
C_0000000002 <|-- C_0000000011
@enduml
```

rendered pic:
![image](https://user-images.githubusercontent.com/12063710/168858724-b156ab07-821a-438d-a638-1fee3ca2635a.png)

**Comment by bkryza** [comment]
@CyCle1024 Thanks for the detailed issue description. I'll into that this week...

If you're in a hurry, you can try to rebuild clang-uml from branch `improve-alias-template-generation`, where I'm currently refactoring template instantiation relationships - I'm hoping to finalize it in the next week or two but I need to resolve a few edge cases...

If all else fails you can add the relationship for now by hand in the `.clang-uml` configuration file in the plantuml section, e.g.:

```yaml
diagrams:
  mydiagram:
    type: class
    #...
    plantuml:
      after:
        - '{{ alias("test1::TestBase<double*>") --|> alias("test1::TestBase<T>") }}'
```

**Comment by bkryza** [comment]
@CyCle1024 Ok thanks - in that case this another issue related to the glob search pattern processing - thanks for pointing it out! I'll look into it nevertheless...

**Comment by bkryza** [comment]
@CyCle1024 This is now fixed since `0.2.0`, there is a test case checking for that:
* https://github.com/bkryza/clang-uml/blob/master/docs/test_cases/t00048.md

**Comment by CyCle1024** [comment]
@bkryza Thx a lot!
